src/main/java/de/qabel/desktop/ui/about/aboutPopup/AboutPopupController.java

```diff
@@ -0,0 +1,40 @@
+package de.qabel.desktop.ui.about.aboutPopup;
+
+import de.qabel.desktop.ui.AbstractController;
+import javafx.application.Platform;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.TextArea;
+import javafx.scene.layout.Pane;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class AboutPopupController extends AbstractController implements Initializable {
+
+    @FXML
+    public TextArea textAreaPopup;
+
+    @FXML
+    public Pane aboutPopup;
+
+    @Override
+    public void initialize(URL location, ResourceBundle resources) {
+    }
+
+    public void btnClosePopup() {
+        hidePopup();
+    }
+
+    public void setTextAreaContent (String content) {
+        textAreaPopup.setText(content);
+    }
+
+    public void hidePopup() {
+        Platform.runLater(() -> aboutPopup.setVisible(false));
```

Comment:
this method (and `showPopup`) are only called as a result of button clicks. So we should already be on the platform thread. I don't think we need a runLater. (Doesn't really hurt though)